openCVLibrary330/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 24
+    compileSdkVersion 26
     buildToolsVersion "25.0.0"
```

Comment:
Should you also update build tools to 26.0.2?

Reply:
I have a separate PR for that, upgrading the build tools [here](https://github.com/farkam135/GoIV/pull/789), upgrading the build tools and dependencies is a safe an non controversial action. 

I thought moving to target Oreo might be more controversial so I split it out

Reply:
Thank you for the explanation!
I prepared #792 to fix the issues that arise when targeting Oreo.
So this can be merged, and also #789!